src/platform/telemetry/providers/cloud/MixpanelTelemetryProvider.ts

```diff
@@ -471,60 +511,17 @@ export class MixpanelTelemetryProvider implements TelemetryProvider {
       void this.initializeComposables()
     }
 
-    let nodeCounts: {
```

Comment:
Moved to https://github.com/Comfy-Org/ComfyUI_frontend/pull/6522/files#diff-b42e99d0b61368229bac7e984b9695021561ddddd9482dfe30bc4390c564100b